project.py

```diff
@@ -192,7 +348,7 @@ def video_search(self, video):
 
 
 def main(inputFile):
-    board = BoardSearcher()
+    board = BoardSearcher(0, 0, 30, 0.70)
```

Comment:
@Foxiy I get that these are the default values but should they still need to be passed here? Honestly, I have no idea what the first two zeros mean just from looking at this line (I know about the other two numbers just because I read the comments on the top). In general it seems to be better to use keyword arguments in cases like this one in order to avoid ambiguity.